extension/content.js

```diff
@@ -51,6 +52,24 @@ function addReleasesTab() {
 	}
 }
 
+function addBlameParentLinks() {
+	let pathname = location.pathname;
+
+	$('.blame-sha').each(function(index, commitLink) {
+		let $commitLink = $(commitLink)
+		let $blameParentLink = $commitLink.clone();
+		let commitSha = commitLink.href.match(/(?<=commit\/)\w+/)[0];
```

Comment:
I could be wrong, but I can't find a source saying JS supports lookbehind at all. If it's working in your browser then I'm wrong. MDN only seems to discuss lookahead.

Why not just get rid of the look around and use `(\/commit\/\w+)`? That makes the group include the `/commit/[sha]`.